app/views/missing/report.php

```diff
@@ -136,7 +138,7 @@
                                         <label for="lname">Last name </label>
                                         <input type="text" class="form-control" placeholder="Enter Last Name" id="lname" name="person[lname]" required /><br>
                                         <label for="dob">Date of birth </label>
-                                        <input type="date" placeholder="Enter your DOB" class="form-control" id="dob" type="date" name="person[dob]" /><br>
+                                        <input type="text" placeholder="Enter your DOB (yyyy-mm-dd)" class="form-control" id="dob" name="person[dob]" pattern="(0?[1-9]|[12][0-9]|3[01])[\/\-](0?[1-9]|1[012])[\/\-]\d{4}" /><br>
```

Comment:
I am unable to add a new missing person report. Below error keeps propping up.
![screenshot from 2015-04-05 03 18 20](https://cloud.githubusercontent.com/assets/1708579/6994767/90a02b72-db42-11e4-81a7-5ce0c2a17978.png)